changelogs/client_server/newsfragments/1875.clarification

```diff
@@ -0,0 +1 @@
+Include the text secure secret storage and sharing, SSSS, and 4S in the Secrets section to make it easier to find.
```

Comment:
This is a bit overly detailed IMHO

```suggestion
Minor clarifications to the "Secrets" section.
```

Reply:
we should be describing what the clarifications are.

```suggestion
Document the acronyms and alternate names for the "Secrets" section.
```